op-challenger/fault/loader.go

```diff
@@ -56,6 +56,8 @@ func (l *loader) fetchClaim(ctx context.Context, arrIndex uint64) (Claim, error)
 			Value:    fetchedClaim.Claim,
 			Position: NewPositionFromGIndex(fetchedClaim.Position.Uint64()),
 		},
+		Countered:           fetchedClaim.Countered,
```

Comment:
I'm slightly nervous about starting to use the `Countered` value from the contract as it's one of the mutable fields so we're breaking our data access plan.  That said, at the moment we're not attempting to cache any game state and its looking likely that we won't need to so this isn't currently going to cause any issues.

Probably just worth a comment to note that we're reading a mutable field from the contract and will need to stop doing that if we ever start caching the game state and only processing new claims rather than starting from scratch each time.